lab_2_retrieval_w_bm25/main.py

```diff
@@ -19,6 +19,20 @@ def tokenize(text: str) -> list[str] | None:
     In case of corrupt input arguments, None is returned.
     """
 
+    if not isinstance(text, str):
+        return None
+    tokens = []
+    word = []
+    for token in text.lower():
```

Comment:
too rude, maybe there are some ways to split text into tokens?

Reply:
Done

Reply:
what done? it is not rude now. Try to use .split() function

Reply:
I used it and it didn't work, now it's fixed